app/src/org/commcare/CommCareApplication.java

```diff
@@ -841,13 +841,14 @@ public void clearUserData() {
         CommCareApplication._().getCurrentApp().getAppPreferences().edit()
                 .putString(CommCarePreferences.LAST_LOGGED_IN_USER, null).commit();
 
-        // manually clear file-backed fixture storage to ensure files are removed
-        CommCareApplication._().getFileBackedUserStorage("fixture", FormInstance.class).removeAll();
-
         CommCareApplication._().closeUserSession();
     }
 
     public void wipeSandboxForUser(final String username) {
+        // manually clear file-backed fixture storage to ensure files are removed
+        CommCareApplication._().getFileBackedUserStorage("fixture", FormInstance.class).removeAll();

```

Comment:
Since the user db was being cleared before this was called I don't think it was doing anything, because the file paths were cleared before this code could remove the files. Hence leaving dangling fixture data (for fixtures over 1mb). Moving it here ensures that files are deleted before the db is cleared.